app/pages/AdviceStartPage.scala

```diff
@@ -16,4 +16,11 @@
 
 package pages
 
-case object AdviceStartPage extends Page
+import play.api.libs.json.JsPath
+
+case class AdviceStartPage(recordId: String) extends QuestionPage[String] {
```

Comment:
Don't think this needs to be a QuestionPage? User enters no answer

can't it just be 

case class AdviceStartPage(recordId: String)

Reply:
tried with just page but it has now recordId in path which isn't allowing me to use Page unless we modify Page.scala itself

Reply:
Should just work as a Page. See e.g. https://github.com/hmrc/trader-goods-profiles-frontend/pull/172/files#diff-c9b5d4c4d6c81750f6d06c5c48617d347aa3ed55fcc65cd3e950e31b9f0480c2

Reply:
now extending with Page